### DISCUSSION TITLE
Different LED indicators for A and B

### DISCUSSION THREAD

**Discussion by jangelor** [egzumer/uv-k5-firmware-custom#340] [Ideas]
On some radios like the Yaesu FT3, band A and B have different color indicators when signals are received. Green for A and blue for B.

I noticed that the led becomes blue during USB charging. Is it possible to use the blue color for band B receiving?

**Comment by AndrewTXJ** [comment]
Interesting, i never thought about this, but it is a useful feature for some situation, but it would be a pain for me to find the right code even if it is possible, i have almost zero knowledge on coding, maybe need others to help, but i will also try if i can find it.

**Comment by gartnl** [comment]
I'd say no. Consider this: during charging the HT is turned off. So it is unlikely the blue led is powered by any GPIO from the processor. However, most charging controllers have an indicator output that might power a (blue) led directly.

**Comment by romgeo2** [comment]
Maybe green LED to blink for one channel?
